testplan/common/utils/match.py

```diff
@@ -131,6 +342,8 @@ def match(
         start_time = time.time()
         end_time = start_time + timeout
         read_mode = "rb"
+        continue_search = True
+        current_position = self.position.position
```

Comment:
You should not look into LogPosition in the LogMatcher code, it should not even know the details of that other than it represent a position by the stratgy